src/blog/medias-and-playlists/index.md

```diff
@@ -0,0 +1,81 @@
+---
+title: An introduction to Playlists and Content in Castmill
+type: Use Case
+description: How to get your your digital signage content onto your screens using playlists.
+author: Aron Kornhall
+twitter: aronkornhall
+date: 2023-10-11
+cover: /blog/medias-and-playlists/img15.png
+avatar: https://pbs.twimg.com/profile_images/1387865689873059842/tAs5CDQ0_400x400.jpg
+tags: post
+---
+
+Whether you have a single digital signage screen or run a global network of screens, you will definitely want to control what content they should show. In this blog post, we will introduce the concept of playlists and how to work with them. A playlist contains all the images and videos that you want to display on a screen. The playlist content loops as long as the playlist is running, but you can have multiple playlists scheduled to play at different times during the day. Keeping your content relevant to your target audience is the key here, and playlists help you do that.
+
+If, for example, your screens are in a restaurant, you will probably have one playlist running in the morning with the breakfast menu, another playlist will run around lunchtime, and additional playlists in the evening and nighttime.
+
+If you run a network of screens across multiple sites, you can use shared playlists to show the same content on all screens, even though they belong to different users. Shared playlists can also be used as a collaboration tool allowing multiple users to work together on a single playlist. In a typical multi-site installation, one centrally managed playlist is used in combination with  additional playlists for each site. Similarly, in the cast of a franchise, a franchisor may use a shared playlist to push out content to all franchisees.
```

Comment:
in the cast of a franchise -> in the case of a franchise

Reply:
Intressant att Grammarly inte fångar detta.